### ISSUE TITLE
Support new Mitsui card

### ISSUE THREAD

**Issue by yhkaplan** [yhkaplan/credit-card-scanner#12]
- from: @shimoju
- Looks possible
- https://www.smbc-card.com/mem/hitotoki/learn/newcard.jsp

**Comment by shimoju** [comment]
This design is called "Visa Quick Read Card Design".

document: https://usa.visa.com/dam/VCOM/download/merchants/New_VBM_Acq_Merchant_62714_v5.pdf

**Comment by Zeynal7** [comment]
Is there any plans for supporting this?

**Comment by yhkaplan** [comment]
@Zeynal7 I don't think I'll have the time very soon, but I'm curious if you've come across any credit cards with this vertical format? I haven't seen any in the wild. Have you?

**Comment by Zeynal7** [comment]
@yhkaplan Yes, some of the cards in my country has vertical written card numbers. So, it is now required for our app to support it as well.
